{"uuid": "034ef327-d1be-40c1-bd45-54037c07e240", "children": ["7cb826fc-dd53-4575-a63c-54bc6f287644"], "befores": [{"name": "take_screenshot_on_failure", "status": "passed", "start": 1728981333073, "stop": 1728981333073}], "afters": [{"name": "take_screenshot_on_failure::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\Users\\KUNAL\\SekelTechAssignments\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\KUNAL\\SekelTechAssignments\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\KUNAL\\SekelTechAssignments\\conftest.py\", line 34, in take_screenshot_on_failure\n    if request.node.rep_call.failed:  # Check if the test failed\n"}, "start": 1728981343672, "stop": 1728981343673}], "start": 1728981333073, "stop": 1728981343674}